server/api/profiles.js

```diff
@@ -72,4 +73,19 @@ router.get("/:public_id", async (req, res) => {
   }
 });
 
+router.get("/applicant/:applicant_id", async (req, res) => {
+  const id = req.params.applicant_id;
+  try {
+    let profile = await getProfileByApplicantId(id);
+
+    if (profile == null) {
+      profile = {};
```

Comment:
This will be an array so the default should be empty square bracket `[]`

Reply:
Changed. Thanks for your assistance.